easybuild/easyconfigs/f/flex/flex-2.5.35-goolf-1.4.10.eb

```diff
@@ -8,7 +8,11 @@ sometimes called a tokenizer, is a program which recognizes lexical patterns in
 toolchain = {'name': 'goolf', 'version': '1.4.10'}
 toolchainopts = {'pic': True}
 
-sources = [SOURCELOWER_TAR_GZ]
-source_urls = ['http://prdownloads.sourceforge.net/%(namelower)s']
+sources = ['flex-2-5-35.tar.gz']
```

Comment:
@migueldiascosta well, that's OK, but you wouldn't want to just bump the `version =` line without also seeing that the `sources` line hardcodes the version too...

Login in easyconfigs should be avoided. We're talking pretty old versions of flex here anyway, but hardcoding things in two places should be avoided.